druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/BaseRocksDBLookupExtractor.java

```diff
@@ -122,6 +125,10 @@ public String apply(@NotNull String key, @NotNull String valueColumn, DecodeConf
                     LOG.error("RocksDB instance is null");
                     LOG.error("Failed to get lookup value from cache. Falling back to lookupService.");
                     serviceEmitter.emit(ServiceMetricEvent.builder().build(MonitoringConstants.MAHA_LOOKUP_GET_CACHE_VALUE_FAILURE + "_" + extractionNamespace.getNamespace(), 1));
+                    String cacheByteString = new String(lookupService.lookup(new LookupService.LookupData(extractionNamespace, key, valueColumn, decodeConfigOptional)));
```

Comment:
We do not have to map the result from Lookup Service. Lookup service calls endpoints on peer and will already returned the mapped value. Remapping is not needed.

Reply:
It looks like LookupService here only works on a simple decodeConfig which can only match a single value (not a whole Map of values).